src/db/agentstate/agentstate.cc

```diff
@@ -26,6 +26,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 namespace persist {
 
+const static std::string DEFAULT_PARAMETERS = "\"preferences\":{\"email\" : \" \", \"telephone\" : \" \", \"organization\" : \" \", \"date\":\"DDMMYYYY\", \"temperature\":\"C\", \"language\":\"en-us\", \"time\":\"24h\"}";
```

Comment:
Should the empty strings (`email` etc) be in fact not empty `"\0"` but a space char?

Reply:
These strings do contains one single space each. While using zero-length strings would be more logical, I simply reused the same exact set of defaults parameters as before.

Reply:
Ok, thanks for the confirmation. Then at least this should not break any expectations of existing code :) and may be addressed later.